frontend/src/metabase/nav/components/DatabasePromptBanner/DatabasePromptBanner.tsx

```diff
@@ -0,0 +1,59 @@
+import { t } from "ttag";
+import type { Location } from "history";
+
+import Link from "metabase/core/components/Link/Link";
+import { useDatabaseListQuery } from "metabase/common/hooks";
+import { useSelector } from "metabase/lib/redux";
+import { getUserIsAdmin } from "metabase/selectors/user";
+import { getIsPaidPlan } from "metabase/selectors/settings";
+
+import {
+  ConnectDatabaseButton,
+  CallToActions,
+  DatabasePromptBannerRoot,
+  Prompt,
+  GetHelpButton,
+} from "./DatabasePromptBanner.styled";
+
+interface DatabasePromptBannerProps {
+  location: Location;
+}
+
+export function DatabasePromptBanner({ location }: DatabasePromptBannerProps) {
+  const isAdmin = useSelector(getUserIsAdmin);
+  const isPaidPlan = useSelector(getIsPaidPlan);
+  const { data: databases = [] } = useDatabaseListQuery({
+    enabled: isAdmin && isPaidPlan,
+  });
+  const onlyHaveSampleDatabase =
+    databases.length === 1 && databases[0].is_sample;
+  const shouldShowDatabasePromptBanner =
+    isAdmin && isPaidPlan && onlyHaveSampleDatabase;
+
+  if (!shouldShowDatabasePromptBanner) {
+    return null;
+  }
+
+  const isOnAdminAddDatabasePage = location.pathname.startsWith(
+    "/admin/databases/create",
+  );
+
+  return (
+    <DatabasePromptBannerRoot
+      role="banner"
+      aria-label={t`Database prompt banner`}
```

Comment:
@mazameli @ranquild I'm not sure this warrant a new translation. The label is only used for the E2E test, (and hopefully for screen readers too).

Please feel free to suggest a better name if it should be translated.

Reply:
I wouldn't add an aria-label here, at least not normally. We can either add `data-testid` or just check for something in the banner.

Reply:
Removed, and use `cy.findAllByRole("banner").first()` to target this banner instead.